### DISCUSSION TITLE
Play YouTube Video inside Blazorise Video component

### DISCUSSION THREAD

**Discussion by mdmontesinos** [Megabit/Blazorise#4871] [Q&A]
Blazorise Video component gets a CORB error when the source is set to a YouTube video URL.

```
Cross-Origin Read Blocking (CORB) blocked cross-origin response https://www.youtube.com/video_id with MIME type text/html. See https://www.chromestatus.com/feature/5629709824032768 for more details.
```

I know that the YouTube API requires an iframe to work properly, but I was wondering if there's any way to play it inside the video component, mainly to take advantage of the nice UI and controls it provides.

Also, the docs for the Video Component, in the Ratio attribute, reference YouTube, so it lead me to think it works somehow. 
_"As dimensions are not available from YouTube via SDK, 16:9 is forced as a sensible default."_

**Comment by stsrki** [comment]
It seems we will need to have more work to make YouTube videos work. TBH, I hadn't event tested it when I first created the Video component. I will see what can be done to make it work.
